aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -1602,6 +1602,110 @@ Tensor& linalg_norm_out(Tensor& result, const Tensor& self, std::string ord, opt
   return linalg_norm_out_impl(result, self, c10::nullopt, ord, opt_dim, keepdim, opt_dtype);
 }
 
+// Numerical or None norms
+Tensor linalg_cond(const Tensor& self, optional<Scalar> opt_ord) {
+  TORCH_CHECK(self.numel() > 0, "linalg_cond is not defined for empty tensors.");
+  TORCH_CHECK(self.dim() >= 2, "Tensor of matrices must have at least 2 dimensions.");
```

Comment:
"cond only supports matrices (dims=2) or batches of matrices (dims=3), but got a tensor with dims=X"

Would this work on a 4D tensor, too?

Reply:
It works for ND, N>2, tensors. 4D tensors are tested to work.